### DISCUSSION TITLE
chatgpt-mirai-qq-bot出了3.0的版本 接口路径好像又有改动

### DISCUSSION THREAD

**Discussion by Nothingness-Void** [linweiyuan/go-chatgpt-api#259] [Q&A]
之前用的接口是 `https://域名/chatgpt/backend-api/`
但是现在填进去就无效了，我看了看报错post的路径是 `/backend-api/v1/accounts/chatgpt-web` 
于是我尝试使用 `https://域名/chatgpt/` 依然不对 希望有大佬帮忙解惑

**Comment by linweiyuan** [comment]
目前这个项目是过 Cloudflare 验证码，然后转发请求的，也就是说，只是把请求转发过去，再转回来

这样的话，如果是符合 ChatGPT 官方网站规范的链接，这个程序可以完美支持

但是如果别的项目自己搞了一套，这个项目就要去手动适配，这是何苦呢

那么只能不支持 3.0 版本了，你去叫他们项目改回来
